google/cloud/firestore_v1/query.py

```diff
@@ -125,9 +131,12 @@ def get(self, transaction=None) -> list:
             transaction
                 (Optional[:class:`~google.cloud.firestore_v1.transaction.Transaction`]):
                 An existing transaction that this query will run in.
-        If a ``transaction`` is used and it already has write operations
-        added, this method cannot be used (i.e. read-after-write is not
-        allowed).
+                If a ``transaction`` is used and it already has write operations
+                added, this method cannot be used (i.e. read-after-write is not
+                allowed).
+            retry (google.api_core.retry.Retry): Designation of what errors, if any,
```

Comment:
Is it possible to expose what default retries exist for a call? Or perhaps a way to modify this to indicate that if not set, there are still retries? That a user doesn't need to specify this information to get retry behavior?

Also, what would this need to be set to to have no retry? It is defaulted to None, would an empty list work?

Reply:
Exposing the actual default retries is infeasible:  the values are [buried in generated code](https://github.com/googleapis/python-firestore/blob/c122e4186808468a2ff82e9cc54b501809519859/google/cloud/firestore_v1/services/firestore/transports/base.py#L110-L300).  We could add language saying that if no value is passed, a system-supplied default will be used.

To allow turning off retries altogether, we would need to have the default be `google.api_core.gapic_v1.method.DEFAULT`, and then add it to the `**kwargs` only if it doesn't match that sentinel.